src/client/driver/command-executors/execute-action-command.js

```diff
@@ -188,7 +161,24 @@ function createAutomation (elements, command) {
     /* eslint-enable indent*/
 }
 
+function executeWaitActionCommand (command, elementAvailabilityTimeout) {
```

Comment:
since it's a separate executor I suppose we should move it to a separate `execute-wait-command.js` file. And we can return just a promise because we don't need `startPromise` in this case